glusterd2/main.go

```diff
@@ -119,6 +121,29 @@ func main() {
 		log.WithError(err).Fatal("Failed to generate local auth token")
 	}
 
+	// Get the Jaeger endpoints from config file
+	jaegerEndpoint := config.GetString("jaegerendpoint")
+	jaegerAgentEndpoint := config.GetString("jaegeragentendpoint")
+
+	// Create the Opencensus Jaeger exporter endpoints to log traces and spans
+	exporter, err := jaeger.NewExporter(jaeger.Options{
+		Endpoint:      jaegerEndpoint,
+		AgentEndpoint: jaegerAgentEndpoint,
+		ServiceName:   gdctx.HostName,
+	})
+	if err != nil {
+		log.WithFields(log.Fields{
+			"error": err.Error(),
```

Comment:
IMO: field should include the component name, or the object details etc, Error msg must be logged as an error message or a warning message.

Reply:
@prashanthpai Please suggest the best way to log this.

Reply:
I will use log.WithError() and chain it with WithFields() as suggested. Please check latest evidence.